.eslintrc.json

```diff
@@ -25,9 +24,8 @@
       "jsx": true
     }
   },
-  "plugins": ["react", "prettier", "tailwindcss"],
```

Comment:
Sorry, this was causing issues. Having linting check prettier standards is pointless since there is already a pre-commit prettier formatter